src/pkg/cli/compose/fixup.go

```diff
@@ -183,3 +181,9 @@ func FixupServices(ctx context.Context, provider client.Provider, project *types
 
 	return nil
 }
+
+func getImageRoot(imageRepo string) string {
```

Comment:
Technically this is the repo
```suggestion
func getImageRepo(imageRepo string) string {
```